packages/web/src/components/molecules/ArtifactsList/ArtifactsListItem.spec.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+
+import {fireEvent, render, screen, waitFor} from '@testing-library/react';
+
+import ArtifactsListItem from './ArtifactsListItem';
+
+describe('ArtifactsListItem', () => {
+  const artifact = {
+    description: 'artifact-description',
+    name: 'artifact-name',
+    size: 123,
+  };
+  const executionId = '123';
+  const testName = 'test-name';
+  const testSuiteName = 'test-suite-name';
+
+  function createAutoResetSpy(): jest.SpyInstance {
```

Comment:
It could return `jest.SpyInstance<typeof fetch>`, so `@ts-ignore` won't be needed. Additionally, initial spy may be something like `jest.fn(() => Promise.reject(new Error('mockImplementation required')))`